keyboards/40percentclub/polyandry/rules.mk

```diff
@@ -0,0 +1,10 @@
+BOOTMAGIC_ENABLE = lite	# Virtual DIP switch configuration(+1000)
+MOUSEKEY_ENABLE = yes	# Mouse keys(+4700)
+EXTRAKEY_ENABLE = yes	# Audio control and System control(+450)
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+NKRO_ENABLE = no	
+SLEEP_LED_ENABLE = no
+AUDIO_ENABLE = no
+RGBLIGHT_ENABLE = no
+DEFAULT_FOLDER = 40percentclub/polyandry/promicro
```

Comment:
```suggestion
# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = no            # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output

DEFAULT_FOLDER = 40percentclub/polyandry/promicro
```
This file should match `data/templates/avr/rules.mk` (except for this board the `MCU` and `BOOTLOADER` values are defined in subfolders, so the corresponding sections can be removed). I also suggest setting `COMMAND_ENABLE = no` (that feature is not really usable on a macropad like this).